docs/architecture/events.rst

```diff
@@ -0,0 +1,44 @@
+Events
+======
+
+.. note::
+    Events are autogenerated and will be created automatically
+    when certain actions are completed, such us creating the order,
+    cancelling fulfillment or completing a payment.
+
+Order events
+------------
+
++--------------------------+----------------------------------------------------------------+
+| Code                     | Description                                                    |
++--------------------------+----------------------------------------------------------------+
+| ``placed``               | An order was placed by the customer.                           |
```

Comment:
Since we expose those as an enum in the API they end up being uppercase so tables like this can be a bit confusing. Should we add a column for values returned by the GraphQL API? Should we switch the backend to also use uppercase values (this would require a migration and might break existing scripts)?

Reply:
For now, we might just add another column here to note down the GraphQL Enum value, and consider transforming all of the choices to uppercase.